docker/docker-compose.override.test.yml

```diff
@@ -31,8 +31,8 @@ services:
       - MYSQL_USER
       - MYSQL_PASSWORD
       - MYSQL_DATABASE=test
-    volumes:
-      - "./config/mysql/dev.my.cnf:/etc/mysql/conf.d/my.cnf"
+    # volumes:
+    #   - "./config/mysql/dev.my.cnf:/etc/mysql/conf.d/my.cnf"
```

Comment:
Might prefer just removing this entirely if they're no longer necessary.

Reply:
Removed the lines in question.